eeps/eep-0050.md

```diff
@@ -134,57 +134,57 @@ exposed?
 Possible APIs for "maps as sets"
 ================================
 
-### 1. Augment the maps module
+1. Augment the maps module
```

Comment:
This was an odd one. This is, at the same time, a list item and a title. I chose to go with list only, since I don't think it affects readability/comprehension, but do let me know.